blacklisted_cidrs.yml

```diff
@@ -0,0 +1,24 @@
+Schema: yaml_cidr
```

Comment:
Do these need to be capitalized? I'd prefer to standardize casing if not, i.e. `schema:` and `schema_version:`

Reply:
This was brought up by iBug as well. I wanted them to be lower case but that caused PyYAML to put them at the end of the file, which wasn't great either. Apparently it is hardcoded to alphabetize the keys on the main level, with no easy workaround. So I grudgingly capitalized them.

Reply:
In that case, can we standardize `Schema_version` to `SchemaVersion`?

Reply:
Is that really an improvement though? Ugh.